ui/src/app/applications/components/utils.tsx

```diff
@@ -466,7 +466,7 @@ export function getOperationType(application: appModels.Application) {
     return 'Unknown';
 }
 
-const getOperationStateTitle = (app: appModels.Application) => {
+export const getOperationStateTitle = (app: appModels.Application) => {
```

Comment:
Since this is exported now, can you rename it to `GetOperationStateTitle` for convention?

Reply:
I don't think this is Javascript convention, only Golang requires a capital case for exported types/functions. 

We've not use capital case convention in Argo CD UI, so I would keep it as is.